### PR TITLE
Strip Chars functions implemented

### PR DESCRIPTION
**Pull request by javaduke**
Addresses https://github.com/databricks/sjsonnet/issues/67 , https://github.com/databricks/sjsonnet/issues/80,

### CODE REVIEW

File: sjsonnet/src/sjsonnet/Std.scala [hunk 1]
```diff
@@ -324,6 +324,17 @@ object Std {
...
     },
+
+    builtin("rstripChars", "str", "chars"){ (ev, fs, str: String, chars: String) =>
+      str.replaceAll("[" + chars + "]+$", "")
+    },
+    builtin("lstripChars", "str", "chars"){ (ev, fs, str: String, chars: String) =>
+      str.replaceAll("^[" + chars + "]+", "")
+    },
+    builtin("stripChars", "str", "chars"){ (ev, fs, str: String, chars: String) =>
+      str.replaceAll("[" + chars + "]+$", "").replaceAll("^[" + chars + "]+", "")
```
Comment (lihaoyi-databricks) [inline sjsonnet/src/sjsonnet/Std.scala:335]: Would I be right to say that this won't work if `chars` contains `]`? I think an approach using `str.dropWhile(!chars.contains(_)).reverse.dropWhile(!chars.contains(_)).reverse` would avoid that problem

Comment (lihaoyi-databricks) [reply sjsonnet/src/sjsonnet/Std.scala:335]: Using `indexWhere(!chars.contains(_))` and `lastIndexWhere(chars.contains(_))` to shorten the string would probably work too, and much faster than `dropWhile` and `takeWhile`. We should also make sure to convert `chars` into a `Set` to ensure fast lookup

Comment (javaduke) [reply sjsonnet/src/sjsonnet/Std.scala]: Ah, you were absolutely right, my bad - but `Regex.quote()` takes care of all special chars :)

**Comment by lihaoyi-databricks** [comment]
Looks reasonable to me I think

**Pull request merged**